python/doc/theory/probabilistic_modeling/dickey_fuller.rst

```diff
@@ -4,34 +4,42 @@ Dickey-Fuller stationarity test
+It assumes that the :math:`X: \Omega \times \cD \rightarrow \Rset` process with
+:math:`\cD \in \Rset`, discretized on the time grid :math:`(t_0, \dots, t_{\sampleSize-1})` writes:
 
 .. math::
     :label: DFmodel
 
     X_t = a + bt + \rho X_{t-1} + \varepsilon_{t}
 
-where :math:`\rho > 0` and where :math:`a` or :math:`b` or both :math:`(a,b)` can be assumed to be equal to 0.
+where :math:`\rho > 0` and where :math:`a` or :math:`b` or both :math:`(a,b)` can be assumed
+to be equal to 0.
 
 The Dickey-Fuller test checks whether the random perturbation at time :math:`t` vanishes with time.
 
 When :math:`a \neq 0` and :math:`b=0`, the model :eq:`DFmodel` is said to have a *drift*.
 When :math:`a = 0` and :math:`b \neq 0`, the model :eq:`DFmodel` is said to have a *linear trend*.
 
-In the model :eq:`DFmodel`, the only way to have stochastic non stationarity is to have :math:`\rho = 1` (if :math:`\rho > 1`,
+In the model :eq:`DFmodel`, the only way to have stochastic non stationarity is to have
+:math:`\rho = 1` (if :math:`\rho > 1`,
 then the process diverges with time which is readily seen in the data).
-In the general case, the Dickey-Fuller test is a unit root test to detect whether :math:`\rho=1` against :math:`\rho < 1`:
+In the general case, the Dickey-Fuller test is a unit root test to detect whether
+:math:`\rho=1` against :math:`\rho < 1`:
 
-The test statistics and its limit distribution depend on the a priori knowledge we have on :math:`a` and :math:`b`.
+The test statistics and its limit distribution depend on the a priori knowledge we have
+on :math:`a` and :math:`b`.
 In case of absence of a priori knowledge on the structure of the model, several
-authors have proposed a global strategy to cover all the sub-cases of the model :eq:`DFmodel`, depending on the possible values on :math:`a` and :math:`b`.
+authors have proposed a global strategy to cover all the sub-cases of the model
+:eq:`DFmodel`, depending on the possible values on :math:`a` and :math:`b`.
 
-The strategy implemented is recommended by Enders (*Applied Econometric Times Series*, Enders, W., second edition, John Wiley \& sons editions, 2004.).
+The strategy implemented is recommended by Enders (*Applied Econometric Times Series*,
+Enders, W., second edition, John Wiley \& sons editions, 2004.).
 
 
 
-We note :math:`(X_1, \hdots, X_n)` the data, by :math:`W(r)` the Wiener process, and :math:`W^{a}(r) = W(r) - \int_{0}^{1} W(r)\di{r}`,
-:math:`W^{b}(r) = W^{a}(r) - 12 \left(r - \frac{1}{2} \right) \int_{0}^{1} \left(s - \frac{1}{2} \right) W(s)\di{s}`.
+We note :math:`(X_1, \hdots, X_{\sampleSize})` the data, by :math:`W(r)` the Wiener process,
+and :math:`W^{a}(r) = W(r) - \int_{0}^{1} W(r)\di{r}`,
+:math:`W^{b}(r) = W^{a}(r) - 12 \left(r - \frac{1}{2} \right) \int_{0}^{1} \left(s -\frac{1}{2} \right) W(s)\di{s}`.
 
 
 **1.** We assume the model :eq:`Model1`:
```

Comment:
I don't understand the difference between `Model1` and `DFmodel`. Generally speaking this page should probably be overhauled, but it is out of the scope of this PR.

Reply:
This is a repetition.